app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -206,7 +206,7 @@ class AdminAuthActivityTest {
         )
       ).perform(
         editTextInputAction.appendText("12345"),
-        pressImeActionButton()
+        pressimeAction()
```

Comment:
why this is modified?

Reply:
It was a mistake on my part which is corrected now.